src/main/java/hw_3/Task6.java

```diff
@@ -0,0 +1,18 @@
+package hw_3;
+
+import java.util.Arrays;
+
+public class Task6 {
+    public static void main(String[] args) {
+        //Створіть 3 змінних таких типів: byte, int, long. Створіть масив типу int. Спробуйте туди покласти змінні.
+        byte a = 12;
+        int b = 210;
+        long c = 3113;
+
+       // int[] = {a, b, c} error, Java does not allow mixing primitive types in arrays directly
+
+        Object[] differentTypes = {a, b, c};
+
```

Comment:
You need to create an int[] array and put all your above values into it. Try to implement it.

Reply:
fixed